permissions/plugin-stackrox-container-image-scanner.yml

```diff
@@ -0,0 +1,6 @@
+name: "stackrox-container-image-scanner"
+github: "jenkinsci/stackrox-container-image-scanner-plugin"
+paths:
+- "org/jenkins-ci/plugins/stackrox-container-image-scanner"
```

Comment:
This path should be your `groupId` with periods replaced with forward slashes and then your `artifactId`. According to this (https://github.com/jenkinsci/stackrox-container-image-scanner-plugin/blob/master/pom.xml#L11) your `groupId` is `com.stackrox.jenkins.plugins`, so the path should be `com/stackrox/jenkins/plugins/stackrox-container-image-scanner`

Reply:
Thanks @slide, looks like the groupId is optional ... so I will have that removed from the pom.xml - in that case the path `org/jenkins-ci/plugins/stackrox-container-image-scanner` should be ok.

Reply:
@clickboo Yes, it is optional, we do prefer to have something there, though this will be fine.